language/tools/move-cli/src/mode.rs

```diff
@@ -0,0 +1,170 @@
+// Copyright (c) The Diem Core Contributors
```

Comment:
In the future we could even consider making the on-disk state view its own crate, since the concept is actually pretty general and could end up being useful to other tools and tests. (Noticed that the transaction replay tool is using it right now and it seems a little bit weird for that to depend on the move-cli crate.)

Maybe we could also merge this with some of the in-meory storage implementations in `move-vm-test-utils`, and make a crate called `move-storage` or `move-mock-storage`?

Reply:
Love this idea. Totally agreed that this has use-cases elsewhere; I would love to be able to debug a VM test by spelunking in an on-disk `storage` directory...